x-pack/plugins/search_indices/public/components/start/hooks/use_indices_redirect.tsx

```diff
@@ -36,9 +39,18 @@ export const useIndicesRedirect = (indicesStatus?: IndicesStatusResponse) => {
     if (indicesStatus.indexNames.length === 1) {
       navigateToIndexDetails(application, http, indicesStatus.indexNames[0]);
       setHasDoneRedirect(true);
+      usageTracker.click(AnalyticsEvents.startCreateIndexCreatedRedirect);
```

Comment:
This will fire when the user only has one index and they click home but might be ok. Just something to be aware of in the code.

Reply:
it won't, the `if` clause above this catches if there are any indices defined the first status we receive and redirects to the list page.

This `if` clause is then only hit when a polling response returns a single index after previously having 0. 

This is basically the clause for when you create an index via dev console or outside the UI while sitting on the page.

Reply:
Got it thanks yeh makes sense now. Its only on the diff of the poll.